locust/runners.py

```diff
@@ -1206,7 +1230,7 @@ def on_report_to_master(client_id: str, data: dict[str, Any]):
         self.environment.events.report_to_master.add_listener(on_report_to_master)
 
         # register listener that sends quit message to master
-        def on_quitting(environment: Environment, **kw) -> None:
+        def on_quitting(environment: Environment, **kw: Any) -> None:
```

Comment:
Is the Any needed? Type hinting stuff that the method never uses is kinda meaningless..